apps/formbricks-com/pages/docs/integrations/zapier/index.mdx

```diff
@@ -0,0 +1,110 @@
+import SubmitTestResponse from "./submit-test-response.png";
+import SuccessConnection from "./success-connected.png";
+import TestSubmission from "./test-submission.png";
+import UpdateQuestionId from "./update-question-id.png";
+import ZapierMessage from "./zapier-message.png";
+
+export const meta = {
+  title: "Zapier Setup",
+  description: "Wire up Formbricks with Zapier and 5000+ other apps",
+};
+
+Zapier is a powerful ally. Hook up Formbricks with Zapier and you can send your data to 5000+ other apps. Here is how to do it.
+
+<Callout title="Nail down your survey first" type="note">
+  Any changes in the survey cause additional work in the Zap. It makes sense to first settle on the survey you
+  want to run and then get to setting up Zapier.
+</Callout>
+
+## Step 1: Setup your survey incl. `questionId` for every question
+
+When setting up the Zap your life will be easier when you change the `questionId`s of your survey questions. You can only do so **before** you publish your survey.
+
+<Image src={UpdateQuestionId} alt="Update Question ID" quality="100" className="rounded-lg" />
+
+_In every question card in the Advanced Settings you find the Question ID field. Update it so that you’ll recognize the response tied to this question._
+
+<Callout title="Already published? Duplicate survey" type="note">
+  You can only update the questionId when the survey was not yet published. Already published it? Just
+  **duplicate it** to update the questionIds.
+  <Image src={DuplicateSurvey} alt="Duplicate Survey" quality="100" className="rounded-lg" />
+</Callout>
+
+## Step 3: Send a test response
+
+In order to set up Zapier you’ll need a test response. This allows you to select the individual values of each response in your Zap. If you want to set up an in-app survey, you can use our [Demo App](/docs/contributing/demo) to trigger a survey and submit a response that without much setting. It also works with Formbricks Cloud.
+
+<Image src={SubmitTestResponse} alt="Submit Test Response" quality="100" className="rounded-lg" />
+
+## Step 4: Setup your Zap
+
+Go to zapier.com and create a new Zap. Search for “Formbricks” to get started:
+
+<Image src={AddNewZap} alt="Add New Zap" quality="100" className="rounded-lg" />
+
+Then, choose the event you want to trigger the Zap on:
+
+<Image src={ChooseEvent} alt="Choose Event" quality="100" className="rounded-lg" />
+
+## Step 5: Connect Formbricks with Zapier
+
+Now, you have to connect Zapier with Formbricks via an API Key:
+
+<Image src={ConnectWithFB1} alt="Connect with Formbricks - 1" quality="100" className="rounded-lg" />
+<Image src={ConnectWithFB2} alt="Connect with Formbricks - 2" quality="100" className="rounded-lg" />
+
+Go back to your Formbricks → Settings → API Keys and create an API key for the environment you’re working with: When you **don’t** see an orange bar at the top of your app, you are in Production Mode hence you need a Production Key:
```

Comment:
I would move this to the `API Key Setup page` and reference/link it on the zapier docs page.

Reply:
makes sense, updated